tasks_hayahammad.md

```diff
@@ -0,0 +1,104 @@
+# First step on command line.
+1. Display your current directory.
+#pwd 
+2. Change to the /etc directory.
+#cd /etc
+3. Now change to your home directory using only three key presses.
+#cd ~ 
+4. Go to the parent directory of the current directory.
+#cd ..
+5. Go to the root directory.
+#cd /
+6. List a long listing of the root directory using two ways.
+1.#ls -l 
+2.#ls -la 
+7. Using man pages list all files under /usr/bin and show list of files in long listing format,human readable size format,and show the oldest files first.
+#ls /usr/bin
+#ls -lh  /usr/bin
+#ls -lrt  /usr/bin
+8. Stay where you are, and list the contents of /bin and /sbin.
+#ls /bin
+#ls /sbin
+9. List the files in /boot in a human readable format.
+#ls -lh /boot
+10. Create in one command the directories ~/dir1/dir2/dir3 (dir3 is a subdirectory
+from dir2, and dir2 is a subdirectory from dir1 ).
+#mkdir -p dir1/dir2/dir3
+11. create /tmp/test-fld folder and /tmp/test-File file and      move /tmp/test-File inside /tmp/test-fld. Now remove /tmp/test-fld using two ways.
+#mkdir /tmp/test-fld
+#touch /tmp/test-File
+#mv /tmp/test-File /tmp/test-fld
+# rm -r /tmp/test-fld
+#rm -rf /tmp/test-fld 
+12. Display the type of file of /bin/cat, /etc/passwd and /usr/bin/passwd.
+#file /bin/cat
+#file /etc/passwd
+#file /usr/bin/passwd 
+13. Display the type of the file /usr/share/man/man1/bc.1.gz
+#file /usr/share/man/man1/bc.1.gz
+14. Create the files today.txt and yesterday.txt by using touch command.
+# touch today.txt
+# touch yesterday.txt
+15. Change the date on yesterday.txt to match yesterday’s date.
+# touch -d "24 hours ago"  yesterday.txt
+16. Display the first 12 lines of /etc/services.
+# head -12 /etc/services
+17. Display the last line of /etc/passwd.
+#tail -1 /etc/passwd
+18. Use cat to create a file named count.txt that looks like this:
+One
+Two
+Three
+Four
+Five
+#cat>count.txt <<stop
+One
+Two
+Three
+Four
+Five
+stop
+#cat count.txt
+19. Use cp to make a backup of count.txt file to cnt.txt.
+#cp count.txt cnt.txt
+20. Use cat to make a backup of count.txt file to catcnt.txt.
+#cat count.txt>cnt.txt
+21. Display catcnt.txt, but with all lines in reverse order (the last line first).
+#tac catcnt.txt
+22. Use cat to create a file named tailing.txt that contains the contents of /etc/profile followed by the contents of /etc/passwd.
+#cat /etc/profile /etc/passwd>tailing.txt
+23. Use cat to create a file named tailing.txt that contains the contents of tailing.txt **followed** by the contents of /etc/passwd.
+# cat tailing.txt /etc/passwd>tailing.txt
```

Comment:
this command will not work and it will only contain the contents of /etc/passwd as it's not possible to have the file as input and output with cat command 
the right answer is `cat /etc/passwd >> tailing.txt`